semantic_model_generator/admin_app/app_utils.py

```diff
@@ -0,0 +1,741 @@
+import os
+from datetime import datetime
+from io import StringIO
+
+import pandas as pd
+import streamlit as st
+
+from semantic_model_generator.data_processing.proto_utils import (
+    proto_to_yaml,
+    yaml_to_semantic_model,
+)
+from semantic_model_generator.generate_model import raw_schema_to_semantic_context
+from semantic_model_generator.protos import semantic_model_pb2
+from semantic_model_generator.protos.semantic_model_pb2 import Dimension, Table
+
+
+SNOWFLAKE_ACCOUNT = os.environ["SNOWFLAKE_ACCOUNT_LOCATOR"]
```

Comment:
throw error if not defined?

Reply:
the app will by default throws error messages on this so i think we should be good